aicssegmentation/workflow/workflow_definition.py

```diff
@@ -20,24 +20,31 @@ class WorkflowDefinition:
 
     name: str
     steps: List[WorkflowStep]
+    from_file: bool
```

Comment:
With inheritance, can we get rid of this field since it's implied depending on which class we use for the workflow definitions?

Reply:
@Aerendel

Reply:
Yes this can and should go away